tests/Benchmarks/System.Text.Utf8/Utf8StringPerf.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Text;
+using System.Text.Utf8;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.System.Text.Utf8
+{
+    public class Utf8StringPerf
+    {
+        [Params(5, 50000)]
+        public int Length;
+
+        [Params(126, 0xD7FF)]
+        public int MaxCodePoint;
```

Comment:
We end up losing the "description" of the input. "Short ASCII string", "Short string" etc. Any way we can get that back?

Reply:
Unfortunately not ;(